agent/agent.go

```diff
@@ -70,6 +73,10 @@ func (a *Agent) Run() error {
 	policyEvalCh := a.setupPolicyManager()
 	go a.policyManager.Run(ctx, policyEvalCh)
 
+	// Launch eval broker and workers.
+	a.evalBroker = policyeval.NewBroker(a.logger.ResetNamed("policy_eval"), 5*time.Minute, 3)
+	a.initWorkers(ctx)
```

Comment:
may want to make this dynamic later, for autoscaling the number of brokers or simply SIGHUP'ing config

Reply:
Yeah, I was trying to decide between a global config or a per-policy value. 

I think per-policy would be better since it can be configured as needed, but global is appealing because you can just set it to your worst-case value.

So probably both? 😄